src/__private_stories__/skin-components-story.tsx

```diff
@@ -71,394 +72,415 @@ type Args = {
+    return (
+        <Stack space={32} dataAttributes={{testid: `components-group`}}>
+            {children}
+        </Stack>
+    );
+};
+
 export const Default: StoryComponent<Args> = ({variant}) => {
     // Only show inverse header when the rest of the screen is default
     const inverseHeader = variant === 'default';
     return (
         <ResponsiveLayout variant={variant}>
             <Box paddingTop={inverseHeader ? 0 : {mobile: 32, desktop: 48}} paddingBottom={16}>
                 <Stack space={32}>
-                    <HeaderLayout
-                        isInverse={inverseHeader}
-                        bleed
-                        header={
-                            <Header
-                                title="Components in different skins"
-                                description="This story is created to verify how some of the components look like when using
+                    <ComponentsGroup>
+                        {/** Header */}
+                        <HeaderLayout
+                            isInverse={inverseHeader}
+                            bleed
+                            header={
+                                <Header
+                                    title="Components in different skins"
+                                    description="This story is created to verify how some of the components look like when using
                         different skins"
-                            />
-                        }
-                        extra={<Placeholder />}
-                        noPaddingY={!inverseHeader}
-                    />
-                    <NegativeBox>
-                        <RowList>
-                            <Row
+                                />
+                            }
+                            extra={<Placeholder />}
+                            noPaddingY={!inverseHeader}
+                        />
+
+                        {/** Row */}
+                        <NegativeBox>
+                            <RowList>
+                                <Row
+                                    title="Title 1"
+                                    subtitle="Subtitle"
+                                    description="Description"
+                                    href="https://www.google.com/"
+                                />
+                                <Row
+                                    title="Title 2"
+                                    subtitle="Subtitle"
+                                    description="Description"
+                                    href="https://www.google.com/"
+                                />
+                            </RowList>
+                        </NegativeBox>
+
+                        {/** BoxedAccordion */}
+                        <BoxedAccordion index={1}>
+                            <BoxedAccordionItem
+                                asset={<IconInvoicePlanFileRegular />}
```

Comment:
I've replaced the image with an icon to avoid unstable tests because of image not loading on time. An issue has been created to investigate this: https://jira.tid.es/browse/WEB-1917